lib/tasks/rebuild_ucf.rake

```diff
@@ -0,0 +1,21 @@
+namespace :freereg do
+
+  desc "Rebuild UCF and deploy"
+  task :rebuild_ucf => [:environment] do
```

Comment:
Yes definitely. Not only this please also point me out If there is any other place where I need to change the names such as methods. I will do the changes and push it again.